src/PuzzleTools.livecodescript

```diff
@@ -12,7 +12,7 @@ end toNumber
 
 function fromNumber pNum
    local tNum
-   put ((pNum - 1) mod 26) + 65 into tNum
+   put (pNum mod 27) + 64 into tNum
```

Comment:
This does not wrap around correctly. For example shifting U by 6 will cause `fromNumber(27)` to be called, which gives 64, (@ in ascii).